Sources/Subprocess/Configuration.swift

```diff
@@ -404,15 +404,15 @@ extension Environment: CustomStringConvertible, CustomDebugStringConvertible {
                 if environmentString.utf8.first == Character("=").utf8.first {
                     continue
                 }
-                #endif  // os(Windows)
+                #endif // os(Windows)
 
                 guard let delimiter = environmentString.firstIndex(of: "=") else {
                     continue
                 }
 
-                let key = String(environmentString[environmentString.startIndex..<delimiter])
+                let key = String(environmentString[environmentString.startIndex ..< delimiter])
```

Comment:
I don't think I like this particular rule - it's pretty common to use ranges without spaces.

Reply:
Updated setting.